.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Do I have to remove this file before merging?

Reply:
:)